client/src/components/layout/ParkList.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react"
+import ParkTile from "./ParkTile.js"
+
+const ParkList = (props) => {
+
+  const [park, setPark] = useState([])
+
+
+  const getParks = async () => {
+    try {
+      const response = await fetch("/api/v1/parks")
+
+      if(!response.ok){
+        const errorMessage = `${response.status} (${response.statusText})`
+        const error = new Error(errorMessage)
+        throw(error)
+      }
+      const body = await response.json()
+
+      setPark(body.parks)
+
+    } catch (error) {
+      console.log(error)
+      console.error(`Error in fetch: ${error.message}`)
+    }
+  }
+  useEffect(() => {
+    getParks()
+  }, [])
+
+  const parksListItems = park.map(parksItem => {
+
+    return(
+    <>
+      <ParkTile key={parksItem.id}
+        {...parksItem}
+      />
+    </>
+    )
+  }) 
+
+  return (
+    <div>
+      <h1>All parks</h1>
+      <ul>
+        {parksListItems}
+      </ul>
+    </div>
```

Comment:
You can see here in this return we are returning multiple elements - the `h1` and the `ul`. Since there's more than one element being returned, we need to wrap them together like with this `div`. We could also use a fragment, but it's not necessary to make that change. Take a look back at this article on Fragments!
https://learn.launchacademy.com/lessons/react-nested-components